client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -714,21 +711,8 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
       }
     };
 
-    $scope.setActiveTab = function (tabNum){
-      if(tabNum === 3){
-      /*Prevent going to third tab if something is invalid in the first tab.
-        Cannot just use $scope.state.OptimizationForm.$invalid for this because the validation of the years and the budgets is done in a different way. */
-        checkValidation();
-        if(errorMessages.length > 0){
-          modalService.informError(errorMessages, 'Cannot view results');
-          return;
-        }
-        if ($scope.state.OptimizationForm.$invalid) {
-          modalService.inform(function() {}, 'Ok', 'Please correct all errors on this page before proceeding.', 'Cannot view results');
-          return;
-        }
-      }
-      $scope.state.activeTab = tabNum;
+    $scope.setActiveTab = function (tabNum) {
+      $scope.activeTab = tabNum;
```

Comment:
@sebastianconcept how did that work for you? activeTab is on state now - I fixed it in a commit